lib/Dancer2.pm

```diff
@@ -196,11 +196,25 @@ how to convert a Dancer (1) based application to Dancer2.
 
 =over
 
+=item * Core and community policy, and standards of conduct
+
+The L<Dancer core and community policy, and standards of conduct> clearly explains
```

Comment:
I think `defines` suits the meaning better than `explains`. And the Strunk in me whispers that the `clearly` can be skipped altogether.

```suggestion
The L<Dancer core and community policy, and standards of conduct> defines
```

Reply:
Bonus points for the Strunk reference!